src/app/components/Menu.tsx

```diff
@@ -101,14 +101,12 @@ export default function Menu() {
           <SvgLogo />
         </a>
       ) : null}
-      {menu.map(({ name, image, link }) => {
+      {menu.map(menuUnitProps => {
         return (
           <MenuUnit
-            key={name}
-            name={name}
-            image={image}
-            link={link}
-            isActive={activeButton === name}
+            {...menuUnitProps}
```

Comment:
There is a way to write a lambda shorter if it only have one return. 
From:
```tsx
menu.map(menuUnitProps => {
  return (
    <MenuUnit
      {...menuUnitProps}
 ```
 
 to:
```tsx
menu.map(menuUnitProps => (
    <MenuUnit
      {...menuUnitProps}
 ```